test/CMakeLists.txt

```diff
@@ -56,6 +56,12 @@ set(SRC
     opexecuter.cpp
 )
 
+if(NGRAPH_TF_ENABLE_VARIABLE_SUPPORT)
+    list(APPEND SRC test_variable_ops.cpp)
+endif()
```

Comment:
The graphs in the test file `test_variable_ops.cpp` should run even if your changes are not in right? (by assigns falling back to TF etc). Maybe we should enable it without the `if(NGRAPH_TF_ENABLE_VARIABLE_SUPPORT)`

Reply:
Yes, they should run.